v21.1/upgrade-cockroach-version.md

```diff
@@ -81,11 +81,23 @@ Verify the overall health of your cluster using the [DB Console](ui-overview.htm
 
 ### Check decommissioned nodes
 
-Check the `membership` field in the [output of `cockroach node status --decommission`](cockroach-node.html). Nodes with `decommissioned` membership are fully decommissioned, while nodes with `decommissioning` membership have not completed the process. If there are `decommissioning` nodes in your cluster, this will block the upgrade.
+If your cluster contains partially-decommissioned nodes, they will block an upgrade attempt.
 
-**Before upgrading from v20.2 to v21.1**, you must manually change the status of any `decommissioning` nodes to `decommissioned`. To do this, [run `cockroach node decommission`](remove-nodes.html#step-2-start-the-decommissioning-process-on-the-nodes) on these nodes and confirm that they update to `decommissioned`.
+1. To check the status of decommissioned nodes, run  the [`cockroach node status --decommission`](node-shutdown.html?filters=decommission#cockroach-node-status) command:
```

Comment:
I went to do this but it's already there. See https://www.cockroachlabs.com/docs/stable/cockroach-node.html#synopsis. The decommission example is the 5th one down.

Reply:
I saw that, but meant a fuller example under the `Examples` heading: https://www.cockroachlabs.com/docs/stable/cockroach-node.html#examples

Reply:
It looked like the decision was made on purpose to link out to the main decommissioning page instead of including that example, especially since decommissioning comes with some risk and nuance. WDYT?

I've fixed the other build issues that prevented the HTML preview from building properly.